lib/html/pipeline/emoji_filter.rb

```diff
@@ -98,6 +104,29 @@ def emoji_filename(name)
           "#{::CGI.escape(name)}.png"
         end
       end
+
+      # Return ancestor tags to stop the emojification.
+      #
+      # @return [Array<String>] Ancestor tags.
+      def ignored_ancestor_tags
+        if context[:ignored_ancestor_tags]
+          DEFAULT_IGNORED_ANCESTOR_TAGS | context[:ignored_ancestor_tags].split(",")
```

Comment:
How about accepting an array for `context[:ignored_ancestor_tags]` instead of a comma delimited string and splitting?